README.md

```diff
@@ -1,5 +1,117 @@
+  exportTask: (data: ExportTaskRequest<TaskParameters>) => Promise<ExportTaskResponse>;
+  getGeometries: (jobId: string) => Promise<TaskGeometry[]>;
+  getEstimatedSize: (params?: GetEstimatedSizeParams) => Promise<number>;
+  getFileSize: (catalogRecordID: string, params: GetFileSizeParams) => Promise<number>;
+}
+```
+most of Export-Interfaces functions uses **[@turf/turf](https://www.npmjs.com/package/@turf/turf)** npm package for geographic data.
+
+---
+
+2. import & implement
+
+```typescript
+import { IExportManager } from '@map-colonies/export-interfaces';
+
+export class ExportManager implements IExportManager {
+    ...
+}
+```
+
+3. implements IExportManager functions
+
+- **exportTask** example:
+```typescript
+export class ExportManager implements IExportManager {
+  public constructor(@inject(SERVICES.LOGGER) private readonly logger: Logger) {}
+
+  public async exportTask(data: ExportTaskRequest<TaskParameters>): Promise<ExportTaskResponse> {
+    this.logger.info({ msg: 'log message' });
+    ...
+    ...
+    ...
+    return await jobId; // uuid format ⇨ '4b3d6bcd-bbfd-4b2d-9x5d-ab8dfbdd4be4'
+  }
+}
+```
+
+---
+
+- **getGeometries** exmaple
+```typescript
+  import { IExportManager, TaskGeometry } from '@map-colonies/export-interfaces';
+  import { MultiPolygon, Polygon } from '@turf/turf'; // ⇨ using '@turf/turf' npm package for Geometry type
+  ...
+  ...
+
+  public async getGeometries(jobId: string): Promise<TaskGeometry[]> {
+    this.logger.info({ msg: 'log message' });
+    ...
+    ...
+    ...
+    const geometry1: Polygon = {
+      coordinates: [
+        [
+          [38.12777806969146, 34.88509668647414],
+          [37.79098758535008, 34.88509668647414],
+          [37.79098758535008, 34.557014824377575],
+          [38.12777806969146, 34.557014824377575],
+          [38.12777806969146, 34.88509668647414],
+        ],
+      ],
+      type: 'Polygon',
+    };
+
+    const geometry2: MultiPolygon = {
+      coordinates: [
+        [
+          [30.10241695788926, 27.097838272461956],
+          [29.991351500386685, 27.0731170944439],
+          [29.991351500386685, 26.949429475478382],
+          [30.08853377570128, 26.776038590550044],
+          [30.310664690705238, 26.98655005029447],
+          [30.10241695788926, 27.097838272461956],
+        ],
+      ],
+      type: 'Polygon',
+    };
+
+    const taskGeometries: TaskGeometry[] = [{ geometry: geometry1, metadata: { maxRes: 0.732 } }, { geometry2 }]; // ⇨ 'metadata' is optional here
+    
+    return await taskGeometries;
+  }
+```
+
+---
+
+- **getEstimatedSize** ⇨ returns the ***estimated*** exported **data** size
+- **getFileSize** ⇨ returns the ***actuall*** exported **data** size
+---
+
+### Event Message
+---
+Export-Interfaces exposes different event message types for the [Export-Management](https://github.com/MapColonies/export-management) usage (expects to receive those message objects in order to handle them while event has occurred)
```

Comment:
example for events?
on what events we need to trigger?

Reply:
fixed , added the expected task events